sonar-scanner-engine/src/main/java/org/sonar/scanner/issue/DeprecatedIssueFilterChain.java

```diff
@@ -35,11 +36,11 @@
   private final List<IssueFilter> filters;
 
   public DeprecatedIssueFilterChain(IssueFilter... filters) {
-    this.filters = ImmutableList.copyOf(filters);
+    this.filters = Collections.unmodifiableList(Arrays.asList((filters)));
```

Comment:
Redundant parentheses.

I'm also wondering if unmodifiability is really important. `Arrays.asList` is already fixed size. Modification with `list.set` would still be possible, but is that a real concern?